### PR TITLE
Fix warnings related to Swift 4.1

### PR DESCRIPTION
**Pull request by AvdLee**
Fixes warnings related to Swift 4.1

### CODE REVIEW

**Review by renssies — APPROVED** [review]

**Comment by ghost** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 10 tests, with 0 failures (0 unexpected) in 0.910 (0.917) seconds</td>
    </tr>
  </tbody>
</table>

## Current coverage for Mocker is `94.62%`
Files changed | - | - 
--- | --- | ---
MockingURLProtocol.swift | `90.48%` | :white_check_mark:
Mock.swift | `95.59%` | :white_check_mark:

---

> Powered by [xcov](https://github.com/nakiostudio/xcov)

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: Sources/MockingURLProtocol.swift [hunk 1]
```diff
@@ -22,6 +22,7 @@ public final class MockingURLProtocol: URLProtocol {
             let response = HTTPURLResponse(url: mock.url, statusCode: mock.statusCode, httpVersion: Mocker.httpVersion.rawValue, headerFields: mock.headers),
             let data = mock.data(for: request)
         else {
+            // swiftlint:disable nslog_prohibted
```
Comment (Boris-Em) [inline Sources/MockingURLProtocol.swift:25]: Why do we keep dead code?

**Review by Boris-Em — CHANGES_REQUESTED** [review]

File: Sources/MockingURLProtocol.swift (hunk 1 above)
Comment (AvdLee) [reply Sources/MockingURLProtocol.swift:25]: This is to ignore the SwiftLint warning generated. Read more about it here: https://github.com/realm/SwiftLint#disable-rules-in-code

**Comment by AvdLee** [comment]
@Boris-Em ready to review again, check my comment

**Review by Boris-Em — APPROVED** [review]

**Pull request merged**